GitCommands/Git/AlphaGitItemStatusComparer.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GitCommands.Git
+{
+    /// <summary>
+    /// Compares <see cref="GitItemStatus"/> objects by <see cref="GitItemStatus.Name"/> and then by <see cref="GitItemStatus.OldName"/> alphabetically.
+    /// </summary>
+    public class AlphaGitItemStatusComparer : Comparer<GitItemStatus>
+    {
+        public override int Compare(GitItemStatus x, GitItemStatus y)
+        {
+            return x.CompareTo(y);
```

Comment:
If this only delegates to `IComparable`, can't this type be removed and `Comparer<GitItemStatus>.Default` used instead?

Alternatively, actually do the comparison here without delegating, so it's explicit. I prefer custom comparers over one-size-might-fit-all IComparable implementations.